src/scenes/Intro.tsx

```diff
@@ -0,0 +1,43 @@
+import Phaser from 'phaser';
+import { render, Text } from 'phaser-jsx';
+
+import { Button } from '../components';
+import { key } from '../constants';
+
+export class Intro extends Phaser.Scene {
+  constructor() {
+    super(key.scene.intro);
+  }
+
+  create() {
+    const { width, height } = this.scale;
```

Comment:
Is this better than `this.cameras.main.centerX` and `this.cameras.main.centerY`?